### PR TITLE
Adds a banner to the home page to feature the tutorials catalog and YT channel

### PR DESCRIPTION
**Pull request by sorvell**
Partially addresses https://github.com/lit/lit.dev/issues/822

### CODE REVIEW

**Review by graynorton — CHANGES_REQUESTED** [review]
Looks good!

I'd like to request a few minor tweaks: unbold the banner text, use bold instead of italics for the links, slightly simplify the wording, as mocked up here:

image

**Comment by justinfagnani** [comment]
I second @graynorton 's comment, and wonder if we want to add a color difference for the links like we have on some other links on the site - just to make it more apparent that they're links.

**Comment by justinfagnani** [comment]
Also, I think this callout is good for all users, whether they're new to Lit - this is new to our site for everyone. So maybe a tweak to the text like:

"📣 Check out our brand new [Tutorials Catalog](https://pr823-16102a4---lit-dev-5ftespv5na-uc.a.run.app/tutorials) and [YouTube Channel!](https://pr823-16102a4---lit-dev-5ftespv5na-uc.a.run.app/youtube)"

Unsure whether we want more text to emphasize that there are multiple tutorials now, or that YT has multiple series.

**Comment by sorvell** [comment]
Updated text and styling. I tried to find another color for the links but didn't see one that made sense. Seems ok as is?

**Comment by graynorton** [comment]
I think the text and text formatting seem fine. One thing I noticed is that the banner doesn’t look great at mobile width when the text breaks. Maybe the fix is as simple as adding `text-align: center`?

**Comment by arthurevans** [comment]
Minor nit, but we don't call it a Tutorials Catalog anywhere except the code. Can we just say Tutorials page? Maybe:

"... newly-expanded Tutorials page and brand-new Lit YouTube Channel"

Of just "brand-new Tutorials page and Lit YouTube Channel", because the Tutorials page really is brand-new.

**Comment by sorvell** [comment]
Argh, just pushed and didn't see this... ok or change?

**Comment by justinfagnani** [comment]
@sorvell yeah, make the change and we'll quick review/deploy
